notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb

```diff
@@ -608,6 +608,12 @@
         "        {metric: values[-1] for metric, values in history.history.items()}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.            try:
</pre><p>Why try-except and not:</p><p> </p><pre class="ql-syntax" spellcheck="false">aiplatform.log_metrics(
&nbsp; &nbsp; &nbsp; &nbsp; {metric: values[-1] for metric, values in history.history.items() if len(values) &gt; 0}
&nbsp; &nbsp; )
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
The value is Nan or infinity.

Reply:
Consider:

```
aiplatform.log_metrics(
        {metric: values[-1] for metric, values in history.history.items() if (not math.isnan(values[-1])) and math.isfinite(values[-1])}
    )
    ```
   or equivalent